### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Rasbin**
Update config for init dev env

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,12 +21,18 @@
...
 ```bash
 yarn install
-NODE_ENV=development node_modules/.bin/webpack-cli --mode development
+yarn build
 ```
 #### Enable debug mode
 - `touch local_settings.py`
 - Inside `local_setting.py`, have `DEBUG = True`
 
+If you experience missing locale, try add locale to your shell script
```
Comment (juyrjola) [inline README.md]: If you experience a missing locale error, you can try configuring the locale:
```
<commands here>
```

File: README.md [hunk 2]
```diff
@@ -21,12 +21,18 @@
...
+
+#### Init Db (if doesnt exist)
```
Comment (juyrjola) [inline README.md]: Should be:
#### Initialize the database (if it doesn't exist)

Comment (Rasbin) [reply README.md]: Fixed in new commit.

File: README.md (hunk 1 above)
Comment (Rasbin) [reply README.md]: Fixed in new commit.

**Comment by Rasbin** [comment]
Hi @juyrjola can you check it now and approve if it looks fine?

**Pull request merged**